arch/arm64/boot/dts/qcom/sdm660-xiaomi-lavender-boe.dts

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Copyright (c) 2024, Barnabas Czeman <trabarni@gmail.com>
+ */
+
+/dts-v1/;
+
+#include "sdm660-xiaomi-lavender-common.dtsi"
+
+/ {
+	model = "Xiaomi Redmi Note 7 (Boe)";
+	compatible = "xiaomi,lavender-boe", "qcom,sdm660";
```

Comment:
This compatible is not documented in https://github.com/sdm660-mainline/linux/blob/qcom-sdm660-6.8.y/Documentation/devicetree/bindings/arm/qcom.yaml#L870 and it will cause regression in dtschema check warnings. In order to avoid adding all variants there maybe let's just keep compatible = "xiaomi,lavender"? Model string may have variant appened IMO, but compatible... not sure :thinking:

Reply:
Or maybe document both now compatibles in qcom.yaml.. if that's what is used upstream for beryllium